portality/tasks/ingestarticles.py

```diff
@@ -286,12 +286,13 @@ def _process(self, file_upload):
 
         ingest_exception = False
         result = {}
+        ids = []
         try:
             with open(path) as handle:
                 articles = xwalk.crosswalk_file(handle, add_journal_info=False) # don't import the journal info, as we haven't validated ownership of the ISSNs in the article yet
                 for article in articles:
                     article.set_upload_id(file_upload.id)
-                result = articleService.batch_create_articles(articles, account, add_journal_info=True)
+                result, ids = articleService.batch_create_articles(articles, account, add_journal_info=True)
```

Comment:
The article IDs are surely added to each article once they are saved?  You don't need to return the ids, I don't think.  If we do have to change the return type of the `batch_create_articles` and the `create_articles` methods, then we should talk about it in a bit more detail first, as the aim of the BLL is to provide a clean, consistent interface to the core, and this response type change feels hacked in.